Apps/Database/src/Migrations/20211118173135_AddFkInUserPreferenceTable.cs

```diff
@@ -23,12 +23,7 @@ namespace HealthGateway.Database.Migrations
     public partial class AddFkInUserPreferenceTable : Migration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
-        {
```

Comment:
If you wanted to maintain the drop you could probably add an if around this with
Schema.Schema(<yourschema>).Table(<yourtable>).Exists()